doc/cabal-package.rst

```diff
@@ -629,19 +629,78 @@ describe the package as a whole:
     and is rather intended as extra metadata for use by third party
     tooling, such as e.g. CI tooling.
 
-    Here's a typical usage example
+    Here's a typical usage example:
 
     ::
 
-        tested-with: GHC == 8.6.3, GHC == 8.4.4, GHC == 8.2.2, GHC == 8.0.2,
+        tested-with: GHC == 9.0.1, GHC == 8.10.4, GHC == 8.8.4,
+                     GHC == 8.6.5, GHC == 8.4.4, GHC == 8.2.2, GHC == 8.0.2,
                      GHC == 7.10.3, GHC == 7.8.4, GHC == 7.6.3, GHC == 7.4.2
 
-    which can (starting with Cabal 3.0) also be written using the more
-    concise set notation syntax
+    The same can be spread over several lines, for instance:
 
     ::
 
-        tested-with: GHC == { 8.6.3, 8.4.4, 8.2.2, 8.0.2, 7.10.3, 7.8.4, 7.6.3, 7.4.2 }
+        tested-with: GHC == 9.0.1
+                   , GHC == 8.10.4
+                   , GHC == 8.8.4
+                   , GHC == 8.6.5
+                   , GHC == 8.4.4
+                   , GHC == 8.2.2
+                   , GHC == 8.0.2
+                   , GHC == 7.10.3
+                   , GHC == 7.8.4
+                   , GHC == 7.6.3
+                   , GHC == 7.4.2
+
+    The separating comma can also be dropped altogether:
+
+    ::
+
+        tested-with:
+          GHC == 9.0.1
+          GHC == 8.10.4
+          GHC == 8.8.4
+          GHC == 8.6.5
+          GHC == 8.4.4
+          GHC == 8.2.2
+          GHC == 8.0.2
+          GHC == 7.10.3
+          GHC == 7.8.4
+          GHC == 7.6.3
+          GHC == 7.4.2
+
+    However, this alternative might
+    `disappear <https://github.com/haskell/cabal/issues/4894#issuecomment-909008657>`__
+    in the future.
+
+
+    Starting with :pkg-field:`cabal-version` 3.0,
+    there are further conveniences.
+    First, a preceding ``,`` is allowed, so a bullet-list style
+    is possible (recommended):
+
+    ::
+
+        tested-with:
+          , GHC == 9.0.1
+          , GHC == 8.10.4
+          , GHC == 8.8.4
+          , GHC == 8.6.5
+          , GHC == 8.4.4
+          , GHC == 8.2.2
+          , GHC == 8.0.2
+          , GHC == 7.10.3
+          , GHC == 7.8.4
+          , GHC == 7.6.3
+          , GHC == 7.4.2
+
+
+    Second, a concise set notation syntax is available:
```

Comment:
Fourth? Additionally?

Reply:
Ah, this was supposed to continue the "First" from features available starting with 3.0.  Maybe the context is lost over the long display.  I could repeat, like
```
Secondly, starting with 3.0, a concise set notation...
```
Better?

Reply:
Oh, no, the original is better. You just make bold assumptions about people's attention spans. :)

I guess a new paragraph for "First" would have helped me, but feel free to ignore.

Reply:
> bold assumptions about people's attention spans. :)

Haha, typical mistake of a writer, this is why it is good to have proof readers!

I changed this to use an enumeration (clarifies structure).